cmd/opampsupervisor/supervisor/templates/bootstrap.yaml

```diff
@@ -1,11 +1,7 @@
 receivers:
-  otlp:
-    protocols:
-      http:
-        endpoint: "localhost:{{.EndpointPort}}"
```

Comment:
Could you remove `.EndpointPort` and any logic to find a free port from `getBootstrapInfo` in `supervisor.go`?